ASS/AWS.py

```diff
@@ -122,13 +140,63 @@ def remove_bucket(self, bucket_name):
             self.logger.info(f"Connect to bucket {bucket_name}")
             s3 = boto3.resource('s3')
             bucket = s3.Bucket(bucket_name)
-            self.logger.info(f"Start deletion of all objects in bucket {bucket}")
+            self.logger.info(f"Start deletion of all objects in bucket {bucket_name}")
             bucket.objects.all().delete()
-            self.logger.info(f"Start deletion of bucket {bucket}")
+            self.logger.info(f"Start deletion of bucket {bucket_name}")
             bucket.delete()
-            self.logger.info(f"Finished deletion of bucket {bucket}")
+            self.logger.info(f"Finished deletion of bucket {bucket_name}")
         except Exception:
             self.logger.error(f"An error occurred while deleting bucket {bucket_name}")
             raise
 
+    def backup_bucket(self, origin_bucket_name, backup_bucket_name):
+        try:
+            self.logger.info(f"Connect to bucket {origin_bucket_name}")
+            s3 = boto3.client('s3')
+            s3_resource = boto3.resource('s3')
+            self.logger.info(f"Start backup of all objects in bucket {origin_bucket_name}")
+            # Get all objects
+            bucket = s3_resource.Bucket(origin_bucket_name)
+            objects = bucket.objects.all()
```

Comment:
`bucket.objects.all() ` is een betere manier dan `Paginator`. Het kan ook overweg met 1000+ objecten en maakt de code ook beter leesbaar.